presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1035,7 +1035,7 @@ protected Type visitQuantifiedComparisonExpression(QuantifiedComparisonExpressio
                     }
                     break;
                 default:
-                    checkState(false, "Unexpected comparison type: %s", node.getComparisonType());
+                    throw new IllegalStateException(format("Unexpected comparison type: %s", node.getComparisonType()));
```

Comment:
No need for string format when it's a single argument at the end (can use `+` with no loss of readability)

Reply:
There are 222 occurrences of `format\("[^"%]*%s",` so I assume this is not a hard rule but more a matter of taste?